python/test/test_api_models/test/1/model.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+import numpy as np
+import triton_python_backend_utils as pb_utils
+
+
+class TritonPythonModel:
+    @staticmethod
+    def auto_complete_config(auto_complete_model_config):
+        inputs = []
+        outputs = []
+        dims = [-1, -1]
+        optional = True
+        config = auto_complete_model_config.as_dict()
+
+        for data_type in pb_utils.TRITON_STRING_TO_NUMPY.keys():
+            type_name = data_type.split("_")[1].lower()
+            input_name = f"{type_name}_input"
+            output_name = f"{type_name}_output"
+            inputs.append(
+                {
+                    "name": input_name,
+                    "data_type": data_type,
+                    "dims": dims,
+                    "optional": optional,
+                }
+            )
+            outputs.append({"name": output_name, "data_type": data_type, "dims": dims})
+
+        for input_ in inputs:
+            auto_complete_model_config.add_input(input_)
+        for output in outputs:
+            auto_complete_model_config.add_output(output)
+
+        auto_complete_model_config.set_max_batch_size(0)
+        if config["platform"] == "decoupled":
```

Comment:
I think it's fine for testing for now - but I think this use of the `platform` field for convenience ("decoupled", "test", etc) may confuse users that look to the tests for examples. Typically platform field is a backend-specific flag with a meaning for that backend. That being said, I thought it was an interesting use to see 🤓

Reply:
The more idiomatic way to provide some arbitrary config value would probably be to use [config parameters](https://github.com/triton-inference-server/tutorials/blob/336a7d713957c4944fb74b88e2f05baaf1a1d0d3/Quick_Deploy/HuggingFaceTransformers/llama7b/config.pbtxt#L6-L9).
```
parameters: {
  key: "decoupled",
  value: {string_value: "True"}
}
```

Reply:
I was looking for exactly this and couldn't find it -